tests/conftest.py

```diff
@@ -29,3 +33,25 @@ def celery_config():
 def create_app():
     """Create app fixture."""
     return create_app
+
+
+@pytest.fixture
+def locations(db):
```

Comment:
Minor, since is for demo: I think this fixture is already available in pytest-invenio 1.2.0?  I do not see it documented but I see it [here](https://github.com/inveniosoftware/pytest-invenio/blob/master/pytest_invenio/fixtures.py#L560).

tho you have some extra stuff in here.

Reply:
Thanks! I didn't know about the fixture.